s2_jv_electric_boogaloo/healthBar.cpp

```diff
@@ -0,0 +1,60 @@
+#include "healthBar.h"
+
+healthBar::healthBar(int baseHealth)
+{
+	if (baseHealth > 0) {
+		this->health = baseHealth;
+	}
```

Comment:
En ce moment, on ne touche jamais a maxHealth alors on pourrait avoir 10 de vie mais un max de 5. Moi je dis qu'on set les deux en même temps.
```suggestion
	if (baseHealth > 0) {
		this->setMaxHealth(baseHealth);
		this->setHealth(baseHealth);
	}
```